### ISSUE TITLE
 @dnd-kit/modifiers is missing react peer dependency

### ISSUE THREAD

**Issue by igordanchenko** [clauderic/dnd-kit#674]
Hi,

This is a minor issue, but I noticed the following yarn warning:

```
yarn up
➤ YN0000: ┌ Resolution step
➤ YN0002: │ @dnd-kit/modifiers@npm:5.0.0 [e8e0e] doesn't provide react (pa887e), requested by @dnd-kit/utilities
```

```
yarn explain peer-requirements pa887e 
➤ YN0000: @dnd-kit/modifiers@npm:5.0.0 [e8e0e] doesn't provide react, breaking the following requirements:

➤ YN0000: @dnd-kit/utilities@npm:3.1.0 [6afc4] → >=16.8.0 ✘
```

It looks like react should be added as peer dependency to @dnd-kit/modifiers package.json

Thanks

**Comment by madeWORKSHOPdev** [comment]
Could this be why I'm getting `TypeError: modifiers.reduce is not a function` ?

**Comment by igordanchenko** [comment]
Fixed in #823
